zebra/zebra_mpls.c

```diff
@@ -462,6 +486,16 @@ static int fec_send(struct zebra_fec *fec, struct zserv *client)
 	return zserv_send_message(client, s);
 }
 
+/*
+ * Set the variable to true/false, according to the
+ * vtysh configuration of "mpls fec resolve-via-recursive"
+ */
+void zebra_mpls_fec_recursive_seamless_enabled(bool status,
+					       struct zebra_vrf *zvrf)
```

Comment:
Add zvrf apis to the zvrf module, not the zebra_mpls module?

Reply:
Reply: thanks will be moved to zvrf module